python/llm/dev/test/run-example-tests.sh

```diff
@@ -0,0 +1,15 @@
+# LLAMA2 example test
+if [ ! -d $ORIGINAL_LLAMA2_PATH ]; then
+    echo "Directory $ORIGINAL_LLAMA2_PATH not found. Downloading from FTP server..."
+    wget -r -nH --no-verbose --cut-dirs=1 $LLM_FTP_URL/${ORIGINAL_LLAMA2_PATH:2} -P $LLM_DIR
+fi
+
+source $CONDA_HOME/bin/activate bigdl-llm-test
+std=$(python python/llm/example/transformers/transformers_int4/llama2/generate.py --repo-id-or-model-path $ORIGINAL_LLAMA2_PATH)
```

Comment:
We may need taskset for better test performance.
Please refer to here: https://github.com/intel-analytics/BigDL/blob/main/python/llm/test/run-llm-inference-tests.sh#L14 :)